### PR TITLE
[integration] drop glue1 related code

### LINKED ISSUE #4795: Remove code that interacts with Glue1 schema
I don't think it makes sense to keep it anymore: its content might be just misleading.

### PR DESCRIPTION
**Pull request by andresailer**
This should drop all code somehow related to glue1. Except for the strings used in the Glue2 and Bdii2Cs code as dictionary keys, which are just as good as any other string.

Fixes #4795 

BEGINRELEASENOTES

*Configuration
CHANGE: Dropped Glue2Only and Glue2URLS options from Bdii2CSAgent, Glue2 is now the only way
CHANGE: dirac-admin-add-resources: drop -g -G options, Glue2 is now the only way
CHANGE: Configuration.Client.Utilities functions getGridCEs, getSiteUpdates no longer take glue2 parameter

*Interfaces
CHANGE: DiracAdmin: dropped getBDII\* functions, which were looking for Glue1 information
CHANGE: removed dirac-admin-bdii-info command

*Core
CHANGE: Removed Core.Utilities.Grid.getBdiiCEInfo function use Utilities.Glue2.getGlue2CEInfo. Also dropped ldapSite, ldapCluster, ldapCE, ldapCEState, ldapCEVOView, ldapService functions

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/Core/Utilities/Glue2.py [hunk 1]
```diff
@@ -24,19 +24,14 @@
...
 from DIRAC.Core.Utilities.List import breakListIntoChunks
+from DIRAC.Core.Utilities.Grid import ldapsearchBDII
 
 __RCSID__ = "$Id$"
 
 
 sLog = gLogger.getSubLogger(__name__)
 
 
-def __ldapsearchBDII(*args, **kwargs):
```
Comment (fstagni) [inline src/DIRAC/Core/Utilities/Glue2.py:34]: That won't be needed anymore?

Comment (andresailer) [reply src/DIRAC/Core/Utilities/Glue2.py:34]: Now we can directly use ldapsearchBDII from Grid.py, no more circular import

**Comment by andresailer** [comment]
Dare I assume the mail comparison test isn't failing because of my changes https://github.com/DIRACGrid/DIRAC/runs/2391362067#step:5:325 ?

**Comment by chrisburr** [comment]
That test has been flaky for some reason, I'll take a look...

**Comment by chrisburr** [comment]
The mail test will be unflakyed by https://github.com/DIRACGrid/DIRAC/pull/5112

**Comment by chrisburr** [comment]
> `/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5110/src/DIRAC/Core/Utilities/Graphs/GraphUtilities.py:docstring of DIRAC.Core.Utilities.Graphs.GraphUtilities.PrettyDateFormatter.__init__:14: WARNING: Unknown interpreted text role "rc".`

Looks like you need to rebase to pick up the docs fix for matplotlib.

**Comment by andresailer** [comment]
Thanks! Rebased.

**Comment by andresailer** [comment]
@sfayer @marianne013 you are using some of these functions I think? Better to mark them first as deprecated?
https://github.com/ic-hep/GridPPDIRAC/blob/9d3e88d89ba36163bb0c8b7a99f65c39b19323e7/ConfigurationSystem/private/AddResourceAPI.py#L6

**Comment by marianne013** [comment]
I think we weaned ourselves off glue1 and if there are any remnants we  should remove them anyway as a lot of stiff seems to be advertising glue2 only, so I think this can go ahead.
@alexanderrichards

**Review by fstagni — APPROVED** [review]

**Pull request merged**